<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alterar_Campo</class>
 <widget class="QDialog" name="Alterar_Campo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALTERAR CAMPO DO LIVRO</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NOME DO LIVRO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NOME_LIVRO">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>AUTOR </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NOME_AUTOR">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TRILHA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TRILHA">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PESQUISAR LIVRO</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NÚMERO DO LVIRO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="INDICE">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ALTERAÇÃO </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="VALOR_ALTERADO">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CAMPO </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CAMPO">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Nome_livro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nome_autor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trilha</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CDU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Editora</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Colecao</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Local</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data_edic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n_exemplar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>data_entrada</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ALTERAR CAMPO</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
